### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by mylastore** [johndatserakis/koa-vue-notes-api#10]
First of all Thank You for sharing this repo, this is the most complete koa stater I could find. 
Can I skip redis setup until deployment ?

**Comment by mylastore** [comment]
One more question I used MongoDB and kind of new to MySQL.
How do I create the required MySQL databases?

**Comment by johndatserakis** [comment]
Hello - thanks for checking out the repo!

As far as disabling `redis` - I would just comment out lines 20-34 of `./src/index.js`. That will disable it so you can worry about it later.

Well, what I suggest doing for the database support is to just use the included `docker` configuration instance.

To use that, first make sure you have docker installed on your system. Docker is really cool because it lets you set up separate little "worlds" for each of your apps.

Then, because I have the config already setup in this project - you just need to run `docker-compose up -d --build` in this repo's root to get the docker backend running. This backend contains the `MySQL` instance for local development in this case. To stop the containers, just run `docker-compose down` and it will shut them down. You'll want to do that at the end of your coding sessions before you commit or anything.

So to summarize - doing all this will basically create a real `MySQL` instance on your machine - although it's kinda running in a virtual machine of course technically - but that doesn't matter. 

This repo is already set up to work with this instance out of the box. You'll have to create and `.env` file (just duplicate and rename the already existing `./example.env` file in the repo to start) which will expose certain variables to the `./knexfile.js` which in turn connects to the db. You can read more about this in the `README.md`.

How about you give that a shot and let me know how it goes.

**Comment by mylastore** [comment]
Thank you for the response, so MySql databases are created automatically with docker?

**Comment by johndatserakis** [comment]
Yea when you run the command I mentioned, docker will stand up containers. In my config, one of those containers is a `MySQL` database.

**Comment by mylastore** [comment]
cool also new to Docker so to rename the db I just go to the docker configuration files.

Thank you sir.

**Comment by johndatserakis** [comment]
Well to rename the database you need to edit the name in the configuration files, yes, but also when viewing the actual database through an MySQL viewer like [Sequel Pro](https://www.sequelpro.com/), that's where you would need to change the name as well.

For now, as you get the environment set up, I'd just use the existing name so you can make sure you can get the app up and running on your machine.
